### DISCUSSION TITLE
macOS host connected to VPN. Rancher unable to connect to servers in the vpn. Change network settings?

### DISCUSSION THREAD

**Discussion by yevon** [rancher-sandbox/rancher-desktop#2740] [Q&A] [answered]
I have a macOS host connected to a VPN using wireguard. Pods that reside in the rancher kubernetes cluster are unable to connect to servers in the vpn. How can I change network settings of the virtual machine created in macOS? This issue doesn't happen on windows and WSL, pods can connect without problems.

**Comment by adamkpickering** [comment]
It's tough to say without knowing more about your setup, but I'd suggest messing around with your routing table. If you can get traffic coming from the RD VM subnet and destined for the servers in the VPN, and vice versa, it might work. But then there's DNS...

You could also create an issue for this - this sounds like something RD should be able to do.

**Comment by yevon** [comment, marked as answer]
Thanks adam! I just created an issue for this in rancher. I've seen others with the same problem in Lime VM github repo on macos. I will try to play with the routing table, but I think that when rancher updates something my changes will be lost.
